plugin-dev/Source/Sentry/Public/SentryEvent.h

```diff
@@ -51,6 +52,70 @@ class SENTRY_API USentryEvent : public UObject, public TSentryImplWrapper<ISentr
 	UFUNCTION(BlueprintCallable, Category = "Sentry")
 	TArray<FString> GetFingerprint() const;
 
+	/** Sets a tag of the event. */
+	UFUNCTION(BlueprintCallable, Category = "Sentry")
+	void SetTag(const FString& Key, const FString& Value);
+
+	/** Gets a tag of the event. */
+	UFUNCTION(BlueprintPure, Category = "Sentry")
+	FString GetTag(const FString& Key) const;
+
+	/** Tries to get a tag of the event. */
+	UFUNCTION(BlueprintPure, Category = "Sentry")
+	bool TryGetTag(const FString& Key, FString& Value) const;
+
+	/** Removes a tag of the event with the specified key. */
+	UFUNCTION(BlueprintCallable, Category = "Sentry")
+	void RemoveTag(const FString& Key);
+
+	/** Sets tags of the event. */
+	UFUNCTION(BlueprintCallable, Category = "Sentry")
+	void SetTags(const TMap<FString, FString>& Tags);
+
+	/** Gets tags of the event. */
+	UFUNCTION(BlueprintPure, Category = "Sentry")
+	TMap<FString, FString> GetTags() const;
+
+	/** Sets context values of the event. */
+	UFUNCTION(BlueprintCallable, Category = "Sentry")
+	void SetContext(const FString& Key, const TMap<FString, FSentryVariant>& Values);
+
+	/** Gets context of the event. */
+	UFUNCTION(BlueprintPure, Category = "Sentry")
+	TMap<FString, FSentryVariant> GetContext(const FString& Key) const;
+
+	/** Tries to get context of the event. */
+	UFUNCTION(BlueprintPure, Category = "Sentry")
+	bool TryGetContext(const FString& Key, TMap<FString, FSentryVariant>& Value) const;
+
+	/** Sets context values of the event. */
+	UFUNCTION(BlueprintCallable, Category = "Sentry")
+	void RemoveContext(const FString& Key);
+
+	/** Sets an extra of the event. */
+	UFUNCTION(BlueprintCallable, Category = "Sentry")
+	void SetExtraValue(const FString& Key, const FString& Value);
+
+	/** Gets an extra of the event. */
+	UFUNCTION(BlueprintPure, Category = "Sentry")
+	FString GetExtraValue(const FString& Key) const;
+
+	/** Tries to get an extra of the event. */
+	UFUNCTION(BlueprintPure, Category = "Sentry")
+	bool TryGetExtraValue(const FString& Key, FString& Value) const;
```

Comment:
could the "Value" suffix be dropped for these as well to make them consistent with the others?

Reply:
The `Value` suffix is now removed.